### PR TITLE
[angle] Fix mingw build

### PR DESCRIPTION
**Pull request by dg0yt**
- #### What does your PR fix?  
  - [x] Fixes dulicate "lib" in name of import libs for libEGL and libGLESv2.
  - [x] Fixes building with mingw.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  no change, no.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  yes

---

Disclaimer: not tested with actual app, just another step towards building Qt.

### CODE REVIEW

**Comment by dg0yt** [comment]
qt5-base doesn't use angle on windows?!

**Review by ras0219 — APPROVED** [review]

File: ports/angle/CMakeLists.txt [hunk 1]
```diff
@@ -679,9 +686,9 @@ if(NOT BUILD_SHARED_LIBS)
     add_definitions("-DANGLE_EXPORT=" "-DANGLE_UTIL_EXPORT=" "-DEGLAPI=" "-DGL_APICALL=" "-DGL_API=")
 endif()
 
-SET_TARGET_PROPERTIES(libANGLE PROPERTIES PREFIX "")
-SET_TARGET_PROPERTIES(libGLESv2 PROPERTIES PREFIX "")
-SET_TARGET_PROPERTIES(libEGL PROPERTIES PREFIX "")
+set_target_properties(libANGLE PROPERTIES PREFIX "")
```
Comment (ras0219) [inline ports/angle/CMakeLists.txt:689]: Shouldn't `libANGLE` be given a similar `IMPORT_PREFIX ""`?

Comment (dg0yt) [reply ports/angle/CMakeLists.txt:689]: `libANGLE` is `STATIC` so there is no import library:

https://github.com/microsoft/vcpkg/blob/a863c84812089836a3c0f2f215ab3e2579fc8acf/ports/angle/CMakeLists.txt#L537

**Comment by dg0yt** [comment]
> qt5-base doesn't use angle on windows?!

@ras0219 Do you have an idea why it is not supported for windows?

https://github.com/microsoft/vcpkg/blob/a863c84812089836a3c0f2f215ab3e2579fc8acf/ports/qt5-base/vcpkg.json#L7-L11

**Comment by Neumann-A** [comment]
> @ras0219 Do you have an idea why it is not supported for windows?

on windows the default is always to use `-opengl dynamic` even for static builds so `angle` is not required for windows. (see https://doc.qt.io/qt-5/windows-requirements.html.)
 `angle` also has been completely removed from Qt6. It is a completely optional runtime dependency which nowadays can commonly be replaced by the normal windows opengl driver. Alternatively software rendering via mesa would also be possible.

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**